# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: browser/app/profile/firefox.js
# Commit: 2a8633af0279
# Full Hash: 2a8633af0279379685ba781c6c469e8b07ec13f4
# Author: Mathew Hodson <mathew.hodson@gmail.com>
# Date: 2022-03-26 09:28:42
# Regressor Bug: 1752853
# File Overlap Count: 1
# Description:
#   Bug 1752853 - Stop using a worker to write session store. r=Gijs,Standard8
#   
#   Remove now unused pref "browser.sessionstore.max_write_failures".
#   Rename and simplify test_shutdown_cleanup.js to make it more clear that
#   it is testing the isFinalWrite option.
# ==============================================================================

diff -r dd81d58d9d5f -r 2a8633af0279 browser/app/profile/firefox.js
--- a/browser/app/profile/firefox.js	Fri Mar 25 19:19:15 2022 +0000
+++ b/browser/app/profile/firefox.js	Fri Mar 25 19:42:36 2022 +0000
@@ -1078,8 +1078,6 @@
 pref("browser.sessionstore.debug", false);
 // Forget closed windows/tabs after two weeks
 pref("browser.sessionstore.cleanup.forget_closed_after", 1209600000);
-// Amount of failed SessionFile writes until we restart the worker.
-pref("browser.sessionstore.max_write_failures", 5);
 
 // Don't quit the browser when Ctrl + Q is pressed.
 pref("browser.quitShortcut.disabled", false);